src/DogStatsd.php

```diff
@@ -155,6 +157,28 @@ public function __construct(array $config = array())
         );
 
         $this->resetTelemetry();
+
+        $originDetectionEnabled = $this->isOriginDetectionEnabled($config);
+        if ($originDetectionEnabled) {
+            $originDetection = new OriginDetection();
+            $containerID = isset($config["container_id"]) ? $config["container_id"] : "";
+            $this->containerID = $originDetection->getContainerID($containerID, $originDetectionEnabled);
```

Comment:
$originDetectionEnabled is always true here, because of the condition. Is there a reason to pass it as a parameter?